sync_radar_data_globus.py

```diff
@@ -148,7 +143,10 @@ def sanity_check(self):
             sys.exit("Data type can only be one of {}. Exiting".format(self.possible_data_types))
```

Comment:
Side note again: you can get parse_args to have conditions where the user has to choose one of the following so that way you don't have to check if it is in the possible data type list. It may even check an integer range :p

Reply:
Excellent, I've updated this to use 'choices' instead of me checking the input